### PR TITLE
Prompt to delete secrets

### PR DESCRIPTION
**Pull request by idrysdale**
If we fail to delete a secret, we log it, but carry on.

### CODE REVIEW

File: fluidkeys/secretreceive.go [hunk 1]
```diff
@@ -87,25 +89,40 @@ func secretReceive() exitCode {
...
 	if sawError {
 		return 1
-	} else {
-		return 0
 	}
+	return 0
 }
 
-func downloadAndDecryptSecrets(key pgpkey.PgpKey) (decryptedSecrets []string, secretErrors []error, err error) {
+func downloadAndDecryptSecrets(key pgpkey.PgpKey) (decryptedSecrets []string, decryptedUUIDs []string, secretErrors []error, err error) {
```
Comment (paulfurley) [inline fluidkeys/secretreceive.go]: Nice for getting this working.

What do you think about this simpler alternative? return `secrets []secret, ...` where `secret` is:
```
type secret struct {
    decryptedContent string
    UUID uuid.UUID
    sentToFingerprint *fingerprint.Fingerprint
}
```

File: fluidkeys/secretreceive.go [hunk 2]
```diff
@@ -190,3 +227,5 @@ type errDecryptPrivateKey struct {
 }
 
 func (e errDecryptPrivateKey) Error() string { return e.originalError.Error() }
+
+type downloadedSecrets map[fingerprint.Fingerprint][]string
```
Comment (paulfurley) [inline fluidkeys/secretreceive.go]: is this still meant to be here?

**Comment by paulfurley** [comment]
Cool! :)

File: fluidkeys/secretreceive.go (hunk 2 above)
Comment (idrysdale) [reply fluidkeys/secretreceive.go]: Good spot, no!

**Pull request merged**